Documentation/security/restrict-pod-access.rst

```diff
@@ -0,0 +1,52 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _security_restrict_pod_access:
+
+========================================
+Restricting privileged Cilium pod access
+========================================
+
+This page shows you how to restrict priveleged access to Cilium pods by limiting access from the Kubernetes API, specifically from `kubernetes exec pod`_.
+
+.. include:: gsg_requirements.rst
+
+Background
+----------
+
+While the Cilium agent no longer requires a specific set of privileges and capabilities, the Cilium agent requires specific privileges to perform essential system and network operations. 
+
+Cilium relies on Kubernetes and Containers to mount filesystems and the environment. Cilium doesn't perform any extra operations that could result in an unsafe volume mount. 
```

Comment:
What does `to mount [...] the environment` mean? `set up`? Or `in` the environment?

Reply:
@tixxdz PTAL ☝🏻

Reply:
@qmonnet yes setup.

What about:
Cilium relies on Kubernetes and Container runtimes to set up the environment and mount the corresponding volumes. It doesn't perform any extra operations that could result in an unsafe volume mount.

Feel free to re-order in case but that's the meaning, and then LGTM for the technical security points.

Reply:
OK thanks, or just adding `set up` (`to mount filesystems and set up the environment`) is fine by me as well, I just felt like there was a verb other than `mount` missing there :)